### ISSUE TITLE
Why does client_golang pull in a second copy of google protobuf?

### ISSUE THREAD

**Issue by ggilley** [prometheus/client_golang#882]
This package has a lot of dependencies. So I was browsing them (using GitHub insights) looking to see what was used. I see this listed as a dependency: 

```
hkspirt / google_protobuf google.golang.org/protobuf 1.26.0-rc.1
```

It seems to be a copy of the google protobuf library with a couple of changes? What's it used for?

**Comment by beorn7** [comment]
I guess this has been pulled in by a requirement of an indirect dependency. It doesn't seem cool to use an RC.

@SuperQ @kakkoyun this happened in #875 . Perhaps you have noticed anything?

**Comment by beorn7** [comment]
Actually, #881 is already out there fixing this.

**Comment by kakkoyun** [comment]
Fixed by #881.
